definitions/features/system_repos.rb

```diff
@@ -0,0 +1,50 @@
+class Features::SystemRepos < ForemanMaintain::Feature
+  metadata do
+    label :system_repos
+    description 'Feature for operations on yum repositories of system'
+  end
+
+  def upstream_repos
+    repositories = {}
+    list.each do |repo, url|
+      upstream_repo_urls.each do |regex|
+        repositories[repo] = url if url =~ regex
+      end
+    end
+    return repositories
+  end
+
+  def list
```

Comment:
This method is returning enabled repos.
It would be good to have more specific name than just mentioning it as a `list`.
example - `enabled_repos_hash or repo_urls_by_repo`.

What do you think?

Reply:
I changed the method name as suggested. Thank You !